### PR TITLE
Submit team ☀️sunshine first blog post

### PR DESCRIPTION
**Pull request by sujinleeme**
🎉Yay!  Hope to enjoy our story! 🎉

cc. @anasofiapinho @gyujincho

### CODE REVIEW

File: blog/_posts/2018-07-19-team-sunshine.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: Team ☀️ 'Sunshine' on RGSoC 2018!
+layout: post
+created_at: Thu Jul 19 2018
+permalink: blog/2018-07-19-team-sunshine.md
```
Comment (v-thakkar) [inline blog/_posts/2018-07-19-team-sunshine.md]: Hi @sujinleeme ! Thanks for the PR. '.md' is not required when it comes to declaring 'permalink' parameter. Please remove it.

File: blog/_posts/2018-07-19-team-sunshine.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+current: blog
+author: Sujin Lee
+twitter: sujinleeme
+categories:
+- student-posts
+---
+
+# Who we are
+<img src="/img/blog/2018/2018-07-19-team-sunshine.jpg" alt="team sunshine photo"/>
```
Comment (v-thakkar) [inline blog/_posts/2018-07-19-team-sunshine.md]: It's preferable to use markdown style to add images [whenever you can] and also use image-credits class to add credits for the images. The comment applies to all the images in the file.

File: blog/_posts/2018-07-19-team-sunshine.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</div>
+
+## Meet shining our supervisor, mentors, and coaches!
+
+<img src="/img/blog/2018/2018-07-19-team-sunshine-group-call" alt="team sunshine with supervisor, coaches and mentors photo"/>
```
Comment (v-thakkar) [inline blog/_posts/2018-07-19-team-sunshine.md]: Along with the common comment for all images in the file, please check the file path and name for this image. It seems to be broken for some reason.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-07-19-team-sunshine.md (hunk 1 above)
Comment (sujinleeme) [reply blog/_posts/2018-07-19-team-sunshine.md]: Solved 5d7288c

File: blog/_posts/2018-07-19-team-sunshine.md (hunk 2 above)
Comment (sujinleeme) [reply blog/_posts/2018-07-19-team-sunshine.md]: Solved: 5d7288c

File: blog/_posts/2018-07-19-team-sunshine.md (hunk 3 above)
Comment (sujinleeme) [reply blog/_posts/2018-07-19-team-sunshine.md]: Solved: 5d7288c

**Review by sujinleeme — COMMENTED** [review]
Hello. @nerdyvaishali! I fixed all issues. Thank you.

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/2018-07-19-team-sunshine.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+permalink: blog/2018-07-19-team-sunshine
+current: blog
+author: Sujin Lee
+twitter: sujinleeme
+categories:
+- student-posts
+---
+
+# Who we are
+![team sunshine photo](/img/blog/2018/2018-07-19-team-sunshine.jpg "team sunshine photo")
```
Comment (v-thakkar) [inline blog/_posts/2018-07-19-team-sunshine.md]: Hi @sujinleeme ! Like I said in my previous review, you also need to add image credits for all images in teh file. You seem to have missed it. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
Thanks @sujinleeme !

**Pull request merged**